sihl-queue/src/repo_sql.ml

```diff
@@ -31,17 +44,18 @@ let job =
           , ( m.tries
             , ( m.next_run_at
               , ( m.max_tries
-                , (m.status, (m.last_error, (m.last_error_at, m.tag))) ) ) ) )
-        ) )
+                , ( m.status
+                  , (m.last_error, (m.last_error_at, (m.tag, Some m.ctx))) ) )
```

Comment:
Please see here for a suggestion on how the context should be handled regarding optionals: https://github.com/oxidizing/sihl/compare/feature/persist-ctx-in-job...mabiede:sihl:feature/persist-ctx-in-job

This might be nicer

Reply:
@mabiede and me had this discussion. I think using `'a list option` instead of `'a list` with `[]` is wrong most of the times, unless you really need a second "empty" value to signal something.

Reply:
Ok I see, in that case it seems fine.